### DISCUSSION TITLE
All my pages are using one custom domain

### DISCUSSION THREAD

**Discussion by NaolGetnet** [community/community#123138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Github is thinking as if my custom domain is a default domain.

**Comment by yoannchaudet** [comment]
This is the expected behavior if you configure a custom domain for your [user/organization site](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites).

**Reply by NaolGetnet** [reply]
How do I change it back to the github.io domain?

**Reply by yoannchaudet** [reply]
Just remove the custom domain you configured for your `NaolGetnet.github.io` repository.

**Comment by NaolGetnet** [comment]
Thank you sir!

**Comment by xsaqlain** [comment]
GitHub may be treating your custom domain as the default domain due to incorrect settings in GitHub Pages or DNS configuration. Double-check and ensure both are configured correctly to prioritize your custom domain. You can solution on this
 https://spotiapkpremium.com

**Comment by xsaqlain** [comment]
GitHub Pages treating your custom domain as the default means it's correctly configured as your site's primary address. Ensure your DNS settings and `CNAME` file are properly set up, and check GitHub Pages settings for any configuration issues. You can visit this blog for detailed step to step guide on how to solve it! https://cheeziousmenuprice.com/

**Comment by charlesedward23400-ui** [comment]
Thanks to inform If you’re at work or just finished and get a bit hungry, go for something fresh and ([healthy](https://thefirehousesubsmenuprice.com/)) instead of just anything. Because when your health is good, everything else in life gets easier to handle.

**Comment by charlesedward23400-ui** [comment]
If you love good food and enjoy trying different sandwiches, desserts, and more every day. then try best [restaurant ](https://thefirehousesubsmenuprice.com/)

**Comment by charlesedward23400-ui** [comment]
Thanks for sharing this awesome post! If you feel a bit hungry during or after work,[ enjoy something fresh](https://thefirehousesubsmenuprice.com/firehouse-loaded-potato-soup/) and tasty. Healthy choices keep your energy up and help you perform your best.
